web/packages/api/src/transfers/toEthereum/pnaFromParachain.ts

```diff
@@ -44,19 +41,14 @@ export class PNAFromParachain implements TransferInterface {
             claimerLocation?: any
             contractCall?: ContractCall
         },
-    ): Promise<DeliveryFee> {
-        const { assetHub, parachain } =
-            "sourceParaId" in source
-                ? {
-                      assetHub: await source.context.assetHub(),
-                      parachain: await source.context.parachain(source.sourceParaId),
-                  }
-                : source
+    ): Promise<DeliveryFeeV2> {
+        const assetHub = await context.assetHub()
+        const parachain = await context.parachain(sourceParaId)
 
         const sourceParachainImpl = await paraImplementation(parachain)
         const { sourceAssetMetadata } = resolveInputs(
             registry,
-            tokenAddress,
+            tokenAddresses[0],
```

Comment:
Same here, should be validated to be exactly 1, otherwise users might send more than 1 asset and the SDK accepts it but doesn't actually use the other tokens.